src/index.ts

```diff
@@ -258,7 +263,7 @@ export const dotBackpackToWallet = async (
     };
   }
 
-  const solanaPublicKeyDetails = publicKeysDetails.find((publicKeyDetails) => {
+  const solanaPublicKeyDetails = publicKeysDetails.find((publicKeyDetails: any) => {
```

Comment:
Likewise, better to strictly type than to use any. But this is old code I can fix it if you like.

Reply:
I added an interface for this but I don't know if it's right

I don't know what it looks like when the api url returns a full array (same as the previous jwt issue)